### PR TITLE
Fix errors in QUICKSTART.md

### PR DESCRIPTION
**Pull request by thisthat**

### CODE REVIEW

File: QUICKSTART.md [hunk 1]
```diff
@@ -231,7 +232,6 @@ public void handle(HttpExchange httpExchange) {
     serverSpan.setAttribute("http.target", "/resource");
     // Serve the request
     ...
-  } finally {
```
Comment (bogdandrutu) [inline QUICKSTART.md:234]: why this? you need finally to ensure that serverSpan is always ended.

**Comment by jkwatson** [comment]
Thanks for updating this. You beat me to it!

File: QUICKSTART.md (hunk 1 above)
Comment (thisthat) [reply QUICKSTART.md]: the span is created inside the try. It is not be visible in the finally block.

Comment (jkwatson) [reply QUICKSTART.md]: good point. the span declaration should happen outside the `try`, I think. And the finally should check for nullity.

Comment (arminru) [reply QUICKSTART.md]: Whoops, sorry! I thought you removed the line accidentally.
Yep, the Quickstart should demonstrate proper usage as described by @jkwatson then.

**Comment by carlosalberto** [comment]
Oops, should have updated that myself.

**Comment by thisthat** [comment]
@carlosalberto  there was still a discussion about the `finally` reverted change by @arminru 
As first thing tomorrow morning, I will provide a follow up with what @jkwatson suggested 😉

File: QUICKSTART.md (hunk 1 above)
Comment (arminru) [reply QUICKSTART.md]: Since this was merged already I fixed it in #1064.

**Comment by thisthat** [comment]
nvm, @arminru is an night owl worker. The discussion is solved with #1064

**Pull request merged**